### PR TITLE
New Language/TextAnalytics SDK

### PR DESCRIPTION
**Pull request by assafi**
## Description

- New TA SDK under a new name `Microsoft.Azure.CognitiveServices.Language.**TextAnalytics**`. This new SDK is the successor of `Microsoft.Azure.CognitiveServices.Language` package which will get it's final update in an upcoming PR (specifying it upcoming deprecation). 
- Initial SDK version is 1.0.1-preview - this is in order to confirm to the previous namespace SDK which final supported version will be 1.0.1-preview and will have the same features added as this one. (i.e. we'll have a version overlap and only a namespace change).  
- New endpoint includes a new TA endpoint (/entities) and expands the Azure region support (+7)

[PR for corresponding swagger spec](https://github.com/Azure/azure-rest-api-specs/pull/2985)

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Please generate the sdk using generate.ps1
Doing this we are able to generate the meta data needed.

**Comment by assafi** [comment]
I did use `generate.ps1` but had to delete the `AssemblyInfo.cs` because VS complained about duplicate attributes.

**Review by dsgouda — CHANGES_REQUESTED** [review]
Since this is a new project in a new directory it should be possible to add an assembly.cs here, let us know if you still run into issues
Also, when the code is generated using the ps1, a log txt file will be generated, please commit this file.

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics.Tests/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests.csproj [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <Description>Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests Class Library</Description>
+    <AssemblyName>Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests</AssemblyName>
+    <Version>1.0.1-preview</Version>
```
[the remark this answers is not in the document]
Comment (assafi) [reply src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics.Tests/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests.csproj]: ok.. although I don't understand what's publishing has to do with it. The version of the test project has changed to confirm to the version of the tested project. If you feel strongly about it I'll change the version number.
(p.s. can anyone add versioning guidelines doc somewhere?)

[the remark this answers is not in the document]
Comment (assafi) [reply src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics.Tests/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests.csproj]: @dsgouda I just ran `generate.ps1` again and I don't see any log file. Where can I find it?

**Review by dsgouda — APPROVED** [review]
LGTM subject to CI builds passing

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.csproj [hunk 2]
```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code above this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.CognitiveServices.Language.TextAnalytics</PackageId>
+    <Description>This client library provides access to the Microsoft Cognitive Services Language APIs.</Description>
+    <VersionPrefix>1.0.1-preview</VersionPrefix>
```
Comment (shahabhijeet) [inline src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.csproj]: @assafi please remove versionPrefix next time. Going to merge it this time.

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**